Integrations/integration-SplunkPy

```diff
@@ -40,12 +42,20 @@ configuration:
   defaultvalue: ""
   type: 13
   required: false
+- display: 'Proxy - in format: localhost:8080'
```

Comment:
usually, the user uses another ip as a proxy..?
if so - it should reflect in the example..